src/Chainweb/Pact/RestAPI/Server.hs

```diff
@@ -125,45 +138,69 @@ somePactServer (SomePactServerData (db :: PactServerData_ v c logger cas))
 
 somePactServers
     :: PayloadCas cas
+    => Logger logger
     => ChainwebVersion
     -> [(ChainId, PactServerData logger cas)]
     -> SomeServer
 somePactServers v =
     mconcat . fmap (somePactServer . uncurry (somePactServerData v))
 
-
-sendHandler :: MempoolBackend ChainwebTransaction -> SubmitBatch -> Handler RequestKeys
-sendHandler mempool (SubmitBatch cmds) =
-    Handler $
+data PactCmdLog
+  = PactCmdLogSend (NonEmpty (Command Text))
+  | PactCmdLogPoll (NonEmpty ByteString)
```

Comment:
I think this needs to not be ByteString.  We're getting `Cannot decode byte '\xb4': Data.Text.Internal.Encoding.decodeUtf8: Invalid UTF-8 stream.` in some places.

Reply:
I pushed a commit fixing this.